resources/views/monitors.blade.php

```diff
@@ -10,48 +10,39 @@
 
 <body>
     <x-menu />
-    <h2>MONITORES CADASTRADOS</h2>
+    <div class="div-text-monitors">
+        <h1>MONITORES CADASTRADOS</h1>
+    </div>
     <div class="box-search">
     <div class="box">
         <svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><!--! Font Awesome Free 6.4.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) Copyright 2023 Fonticons, Inc. --><style>svg{fill:#391959}</style><path d="M416 208c0 45.9-14.9 88.3-40 122.7L502.6 457.4c12.5 12.5 12.5 32.8 0 45.3s-32.8 12.5-45.3 0L330.7 376c-34.4 25.2-76.8 40-122.7 40C93.1 416 0 322.9 0 208S93.1 0 208 0S416 93.1 416 208zM208 352a144 144 0 1 0 0-288 144 144 0 1 0 0 288z"/></svg>
         <form action="{{ route('get.monitors') }}" method="get">
-            <input type="search" name="search" placeholder="Search">
+            <input type="search" placeholder="Search">
```

Comment:
Se retirar o parâmetro name, com o valor "search", a busca não funcionará.
```suggestion
            <input type="search" name="search" placeholder="Search">
```